This patch will add times to your "turn off screen after X minutes" options under screen & lock

Index: /usr/palm/applications/com.palm.app.screenlock/app/controllers/securityconfig-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.screenlock/app/controllers/securityconfig-assistant.js
+++ /usr/palm/applications/com.palm.app.screenlock/app/controllers/securityconfig-assistant.js
@@ -108,7 +108,7 @@ var SecurityconfigAssistant = Class.crea
 	timerModel: {currentTimer: 30},
 	
 	//Available Timer Values.
-	availableTimers: [{label: $L('30 seconds'), value:30},{label: $L('1 minute'), value:60},{label: $L('2 minutes'), value:120},{label: $L('3 minutes'), value:180}],
+	availableTimers: [{label: $L('30 seconds'), value:30},{label: $L('1 minute'), value:60},{label: $L('2 minutes'), value:120},{label: $L('3 minutes'), value:180},{label: $L('5 minutes'), value:300},{label: $L('Never'), value:10800}],
 	
 	//Model for Security Selection List
 	securityModel: {currentSecurity: 'none'},
